lib/rex/post/meterpreter/ui/console/command_dispatcher/core.rb

```diff
@@ -52,6 +52,7 @@ def commands
       "machine_id" => "Get the MSF ID of the machine attached to the session",
       "quit"       => "Terminate the meterpreter session",
       "resource"   => "Run the commands stored in a file",
+      "uuid"       => "Get the UUID for the current session",
```

Comment:
In some odd scenarios, this might conflict with ```sessions -l -v```, but I see the use of having the UUID here as well.